<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>783</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>1131</width>
      <height>731</height>
     </rect>
    </property>
    <widget class="QWidget" name="dms">
     <widget class="QLineEdit" name="fileNameLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>281</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fileNameLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>260</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>File name</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileCategoryLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>310</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
     <widget class="QComboBox" name="categoryComboBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>281</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fileTagsLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>360</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tags</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileTagsLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>380</y>
        <width>281</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileOwnerLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>430</y>
        <width>281</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="fileOwnerLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>410</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Owner</string>
      </property>
     </widget>
     <widget class="QLabel" name="versionLabel">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>410</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileVersionLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>430</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileSizeLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>380</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="sizeLabel">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>360</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="fileCreatedTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>330</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fileCreatedLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>310</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Created</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="fileLastModifiedTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>280</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="fileLastModifiedLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>260</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Last modified</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addDummyFileButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>490</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>addDummyFile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="savePushButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>490</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save to vault</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>511</width>
        <height>731</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="db">
     <widget class="QTableView" name="databaseTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>911</width>
        <height>711</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QComboBox" name="tableComboBox">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>50</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>files</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>file_metadata</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>file_access</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>file_tags</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>file_tag_mapping</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="tableLabel">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>30</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Table</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuButtonSystem">
    <property name="title">
     <string>Source</string>
    </property>
    <addaction name="actionLocal"/>
    <addaction name="actionVault"/>
    <addaction name="actionOther"/>
   </widget>
   <widget class="QMenu" name="menuViw">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDms"/>
    <addaction name="actionDatabase"/>
   </widget>
   <addaction name="menuButtonSystem"/>
   <addaction name="menuViw"/>
  </widget>
  <action name="actionLocal">
   <property name="text">
    <string>Local</string>
   </property>
  </action>
  <action name="actionOther">
   <property name="text">
    <string>Other</string>
   </property>
  </action>
  <action name="actionVault">
   <property name="text">
    <string>Vault</string>
   </property>
  </action>
  <action name="actionDms">
   <property name="text">
    <string>Documents</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
